### ISSUE TITLE
Mention of importlib.metadata.PackageNotFoundError in docs

### ISSUE THREAD

**Issue by tmct** [python/cpython#116981]
# Documentation

importlib.metadata functions such as version() can throw PackageNotFoundError, which doesn't appear on the Python docs search.

I've seen code in the past where this is caught, and it feels like a public API - would you like to see it mentioned somewhere on the page https://docs.python.org/3/library/importlib.metadata.html?

Thank you

**Comment by hugovk** [comment]
Yes, it's exposed in `__all__` so can be considered a public API:

https://github.com/python/cpython/blob/408e127159e54d87bb3464fd8bd60219dc527fac/Lib/importlib/metadata/__init__.py#L31-L44

Would you open a PR to document it?

**Comment by terryjreedy** [comment]
I verified that there is no entry in the metadata doc chapter, and consequently none in the index.  @hugovk, where in the chapter should it go?

**Comment by johnslavik** [comment]
Duplicate of GH-110937
